docs/rules/forbid-component-props.md

```diff
@@ -35,10 +35,14 @@ The following patterns are not considered warnings:
 
 ```js
 ...
-"forbid-component-props": [<enabled>, { "forbid": [<string>] }]
+"forbid-component-props": [<enabled>, { "forbid": [<string>], "forbidPatterns": [<string>] }]
 ...
 ```
 
 ### `forbid`
 
 An array of strings, with the names of props that are forbidden. The default value of this option is `['className', 'style']`.
+
+### `forbidPatterns`
+
+An array of regular expressions as strings, with the patterns of props that are forbidden. e.g: `['^data-']`
```

Comment:
Accepting user string input into regexes is a bad idea. I'd prefer we accept a limited subset, like just *